nuxt.config.js

```diff
@@ -21,11 +21,32 @@ export default {
       { name: "viewport", content: "width=device-width, initial-scale=1" },
       // bing indexing
       { name: "msvalidate.01", content: "E04DE33CC93C0FF892248C9E70A9A918" },
+      {
+        hid: "og:site_name",
+        property: "og:site_name",
+        content: "WhatsAnalyze",
+      },
+      {
+        hid: "og:title",
+        property: "og:title",
+        content: "Analyze your WhatsApp Chats",
+      },
+      {
+        hid: "og:description",
+        property: "og:description",
+        content:
+          "Learn who writes the most and many more! You can even generate a PDF from your chat!",
```

Comment:
```suggestion
          "America's Most Popular WhatsApp Analyzer ✓ Now offering Group chats ✓ Reveal your friends character ✓ No Chat Data is sent to a Server. Get Started now!"
```

Isnt this our standard description?

Reply:
Sebi meint keine meta tags mehr in die nuxt.Config